packages/usa-icon-list/src/content/usa-icon-list.json

```diff
@@ -3,21 +3,24 @@
     {
       "icon": {
         "name": "check_circle",
-        "color": "green"
+        "color": "green",
+        "title": "Yes"
```

Comment:
I know there was some discussion on what was considered "meaningful" text and how these read to the screen reader. Would it be better to use more descriptive text pertaining to the icon rather than the "meaning" ? 

Maybe we can use "green check / red x" instead of yes/no to help prevent confusion in this example? Not sure if it's the best path forward but figured I'd bring it up for discussion